### PR TITLE
Bring back the form honeypot

### PR DESCRIPTION
**Pull request by jerivas**
## Description
Reverts #305

### CODE REVIEW

**Comment by jerivas** [comment]
[I want better spam filtering on the OddSite contact form(s).](https://trello.com/c/BF3I4uKi/217-i-want-better-spam-filtering-on-the-oddsite-contact-forms)

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
+{
```
Comment (jerivas) [inline .devcontainer/devcontainer.json:1]: Snuck this one in, but I think not many people use it, so I'm ok with removing it as well

File: content/_includes/contact.macros.njk [hunk 2]
```diff
@@ -61,6 +62,11 @@ params:
     required=true
   ) }}
 
+  <label data-honey aria-hidden="true">
+    <span>Don't fill out this field:</span>
+    <input name="subject" tabindex="-1" />
```
Comment (jerivas) [inline content/_includes/contact.macros.njk:67]: The `tabindex=-1` avoids people focusing on nothing as they `TAB` through the form. I hope that doesn't indicate to the bots that they should avoid this field 😅

Comment (mirisuzanne) [reply content/_includes/contact.macros.njk:67]: I hope so too!

**Review by mirisuzanne — APPROVED** [review]

**Review by jgerigmeyer — APPROVED** [review]

**Pull request merged**